mmv1/third_party/tgc_next/test/assert_test_files.go

```diff
@@ -198,7 +198,7 @@ func testSingleResource(t *testing.T, testName string, testData ResourceTestData
 			if diff := cmp.Diff(
 				asset.Resource,
 				roundtripAsset.Resource,
-				cmpopts.IgnoreFields(caiasset.AssetResource{}, "Version", "Data", "DiscoveryDocumentURI"),
+				cmpopts.IgnoreFields(caiasset.AssetResource{}, "Version", "Data", "Location", "DiscoveryDocumentURI"),
```

Comment:
'Location' in Image asset is 'us', which is not a standard location and cannot be converted from the Terraform plan json data.